components/radio-button/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "coherent-gameface-radio-button",
-  "version": "3.1.0",
```

Comment:
I think it is not a patch version here and it should be a minor version because it is an enhancement/feature and not a fix.

`"version": "3.2.0"`